devel/pcre/Portfile

```diff
@@ -65,6 +66,10 @@ platform darwin 8 {
     }
 }
 
+if {${build_arch} eq "arm64"} {
+    configure.args-delete --enable-jit
+}
```

Comment:
This will not be correct for the universal variant. This will disable jit for all archs (arm64 and x86_64) when building on an arm64 machine. Similarly, it will not disable jit for any archs (arm64 or x86_64) when building on an x86_64 machine. To be able to pass different flags for different archs you'll have to use the muniversal portgroup, but that can have other consequences, so test it carefully.